assets/chezmoi.io/docs/user-guide/manage-different-types-of-file.md

```diff
@@ -71,6 +71,20 @@ example using `sed`.
     will be empty and it is the script's responsibility to write a complete
     file to the standard output.
 
+`modify_` scripts that contain the string `chezmoi:modify-template` are executed
+as templates with the current contents of the file passed as `.` and the result
```

Comment:
I was thinking if the `.chezmoi` and other data attributes could not be available as well during this template execution.

Reply:
Good point. I'll update the PR.

Reply:
Maybe we could use something like `.chezmoi.stdin`?

Reply:
Done.